validator/client/validator.go

```diff
@@ -980,18 +980,17 @@ func (v *validator) PushProposerSettings(ctx context.Context, km keymanager.IKey
 		return err
 	}
 	log.Infoln("Successfully prepared beacon proposer with fee recipient to validator index mapping.")
-	failedRegistrationPubkeys := make([]string, 0)
+	failedRegistrationCount := 0
 	for _, request := range registerValidatorRequests {
 		// calls beacon API but used for custom builders
 		if err := SubmitValidatorRegistration(ctx, v.validatorClient, v.node, km.Sign, request); err != nil {
 			// log the error and keep going
-			hexKey := hexutil.Encode(request.Pubkey)
-			failedRegistrationPubkeys = append(failedRegistrationPubkeys, hexKey)
-			log.Warnf("failed to register validator for custom builder for %s, error: %v ", hexKey, err)
+			failedRegistrationCount++
+			log.Warnf("failed to register validator for custom builder for %s, error: %v ", hexutil.Encode(request.Pubkey), err)
 		}
 	}
-	if len(failedRegistrationPubkeys) != 0 {
-		return errors.Errorf("Register Validator requests failed for the following publickeys: %v", failedRegistrationPubkeys)
+	if failedRegistrationCount != 0 {
+		return errors.Errorf("Register validator requests failed %v times out of %v requests", failedRegistrationCount, len(registerValidatorRequests))
```

Comment:
nitpick: it's nice to use the typed string formatters (eg `%d` for numbers, `%s` for strings) rather than `%v` where possible, because linters understand them and will complain when types are misaligned.